src/Ev.ServiceBus/Management/Factories/MessageSenderFactory.cs

```diff
@@ -41,10 +40,7 @@ public IMessageSender CreateSender(ClientOptions[] senderOptions)
         }
 
         if (_options.Value.Settings.Enabled == false)
-        {
-            _logger.LogInformation("[Ev.ServiceBus] Initialization of sender client '{ResourceId}': Client deactivated through configuration", options.ResourceId);
```

Comment:
Please keep this log. 
it is only produced during initialization of the app, so the cost of having them is low.
And they are useful to know exactly which resource are initialized